Source/WebCore/page/scrolling/mac/ScrollerPairMac.mm

```diff
@@ -147,9 +147,14 @@ - (void)scrollerImpPair:(NSScrollerImpPair *)scrollerImpPair updateScrollerStyle
 {
```

Comment:
Should we assert that we're on the main thread here?

Reply:
Right, I think we should.

Reply:
Actually I'm seeing this assert fail in the tests. This destruction is coming from the ScrollingTree being destructed. Does it make sense for this to happen off the main thread? Regardless we have the explicit bounce to main thread call here.